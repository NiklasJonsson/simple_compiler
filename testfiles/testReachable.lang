int main() {
	fun();
	fun1();
}
int fun() {
	fun1();
	fun2();
}
int fun2() {
	fun();
}
int fun1() {
	fun1();
}