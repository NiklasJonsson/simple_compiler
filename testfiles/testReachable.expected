main:
	fun
	fun1
	fun2
fun:
	fun
	fun1
	fun2
fun2:
	fun
	fun1
	fun2
fun1:
	fun1
